source/Halibut.Tests/UsageFixture.cs

```diff
@@ -57,19 +56,41 @@ public async Task OctopusCanSendMessagesToTentacle_WithEchoService(ServiceConnec
         }
 
         [Test]
-        [TestCaseSource(typeof(ServiceConnectionTypesToTest))]
-        public async Task OctopusCanSendMessagesToTentacle_WithEchoService_AndAProxy(ServiceConnectionType serviceConnectionType)
+        [TestCase(ServiceConnectionType.Polling, null, null)]
+        [TestCase(ServiceConnectionType.Polling, PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417, null)]
+        [TestCase(ServiceConnectionType.Polling, null, PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417)]
+        [TestCase(ServiceConnectionType.Listening, null, null)]
+        [TestCase(ServiceConnectionType.Listening, PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417, null)]
+        [TestCase(ServiceConnectionType.Listening, null, PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417)]
+#if SUPPORTS_WEB_SOCKET_CLIENT
+        // Polling Over WebSockets does not appear to support a proxy and ignores the ProxyDetails
+        [TestCase(ServiceConnectionType.PollingOverWebSocket, null, null)]
```

Comment:
Should we not have this test case?

Reply:
I left it there in case support gets added e.g. does it still work with a proxy if support is added! Now = yes (but it doesn't use it) Future = unknown. It may be a bit misleading though, so I can remove these and leave a note that WebSocket do not currently use Proxies